Frameworks/CoreGraphics/CGContextCairo.mm

```diff
@@ -1629,6 +1630,229 @@
     UNLOCK_CAIRO();
 }
 
+void CGContextCairo::_CGContextDrawStrokedPath(cairo_path_t* flatPath, int i, int prev, int total, CGFloat lineWidth) {
```

Comment:
This is an internal implementation, I think. Are we sure that the callers adequately ensure it's non-NULL/